spec/data-types-and-values.html

```diff
@@ -69,16 +69,16 @@ <h1>The Tuple Type</h1>
       <p>The Tuple type is the set of all finite and ordered sequences of ECMAScript primitive values including Record and Tuple. Each tuple value holds an associated [[Sequence]] List which is a list of primitive values. The [[Sequence]] List is integer-indexed. The [[Sequence]] List and its values are never modified.</p>
 
       <emu-clause id="sec-immutable-types-tuple-tostring">
-        <h1>Tuple::toString ( _argument_, [ _separator_ ] )</h1>
-        <p>The abstract operation Tuple::toString takes arguments _argument_ (a Tuple), and _separator_ (a String). It performs the following steps when called:</p>
+        <h1>TupleToString ( _argument_, [ _separator_ ] )</h1>

```

Comment:
This algorithm is never called with a separator. Probably it shouldn't take that as a parameter, and just call Array.prototype.join with no arguments.